packages/mds-attachment-service/service/manager.ts

```diff
@@ -10,7 +11,10 @@ export const AttachmentServiceManager = RpcServer(
     onStop: AttachmentServiceProvider.stop
   },
   {
-    name: args => AttachmentServiceProvider.name(...args)
+    name: args => AttachmentServiceProvider.name(...args),
+    writeAttachment: args =>
+      AttachmentServiceProvider.writeAttachment(...((args as unknown) as [SerializedBuffers<Express.Multer.File>])),
```

Comment:
This is a tough one. Not sure there is a great solution. Might be best to just make the argument a `SerializedBuffers<Express.Multer.File>` in the service interface rather than trying to mask the fact that the codec is doing a weird transform for `Buffer` arguments and then just call `JSON.stringify` on the buffer when invoking the client so that the transform is explicit.